src/views/RecIM/components/List/Listing/index.jsx

```diff
@@ -97,11 +93,13 @@ const ActivityListing = ({ activity }) => {
             </Grid>
           </Grid>
           <Grid item container xs={12} sm={7} direction="column" spacing={1}>
-            <Grid item>
-              <Typography sx={{ color: 'gray', fontWeight: 'bold' }}>
-                ActivityStart - ActivityEnd
-              </Typography>
-            </Grid>
+            {activity.StartDate && (
```

Comment:
```suggestion
            {activity.StartDate && activity.EndDate && (
```
Since both are nullable, I don't know if we should be just checking for a single one